hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -200,14 +202,16 @@ module top_${top["name"]} #(
 % endif
 
   // Clock assignments
-% for clock in top['clocks']:
-  % if clock['name'] != "usb" :
-  assign ${clock['name']}_clk = clk_i;
-  % endif
+% for clock_group in top['clock_groups']:
+  % for k in clock_group['clocks']:
+    % if k != "clk_peri_usb" :
+  assign ${k} = clk_i;
```

Comment:
This is temporary , right? Can you add TODO here?

Reply:
yeah that's right. Eventually should be absorbed into clock controller.

Reply:
comment added.